shared/dataflow/codeql/dataflow/VariableCapture.qll

```diff
@@ -391,16 +391,6 @@ module Flow<InputSig Input> implements OutputSig<Input> {
       msg = "ClosureExpr has no body" and not ce.hasBody(_)
     }
 
-    query predicate closureAliasMustBeLocal(ClosureExpr ce, Expr access, string msg) {
```

Comment:
Perhaps, instead of deleting this check, could we update it to check that all aliases are either local or in a nested scope? Since that's what we now rely on.